packages/addons/sveltekit-adapter/index.ts

```diff
@@ -9,13 +9,12 @@ type Adapter = {
 };
 
 const adapters: Adapter[] = [
-	{ id: 'auto', package: '@sveltejs/adapter-auto', version: '^4.0.0' },
-	{ id: 'node', package: '@sveltejs/adapter-node', version: '^5.2.11' },
+	{ id: 'auto', package: '@sveltejs/adapter-auto', version: '^5.0.0' },
```

Comment:
we might want to merge and release https://github.com/sveltejs/kit/pull/13661 before this one